tests/10.ABI.ts

```diff
@@ -125,7 +125,6 @@ describe('ABI type checking', () => {
       '[][][]',
       'stuff[]',
       // static array
-      'ufixed32x10[0]',
```

Comment:
I'd just confirm that something like `bool[01]` also fails for the regex.

Reply:
we only allow `bool[0]` and `bool[1]`, but not `bool[01]`, see comment here: https://github.com/algorand/js-algorand-sdk/issues/497#issuecomment-1013530294

Reply:
Right, I think `bool[01]` should be added here just to check that it correctly throws an error (since it is not allowed).

Reply:
got it got it, that's a good point, will add testcase to exclude this case

Reply:
just add new testcase here: [c4d93ec](https://github.com/algorand/js-algorand-sdk/pull/698/commits/c4d93ececf418b579ea9c173054c20e4a5acb2ac)